Sensus.Shared/UI/ProtocolsPage.cs

```diff
@@ -311,55 +414,55 @@ public ProtocolsPage()
 
                             // pop up wait screen while we get the participation reward datum
                             IEnumerable<InputGroup> inputGroups = await SensusServiceHelper.Get().PromptForInputsAsync(
-                                null,
-                                new InputGroup[] { new InputGroup { Name = "Please Wait", Inputs = { new LabelOnlyInput("Retrieving participation information.", false) } } },
-                                cancellationTokenSource.Token,
-                                false,
-                                "Cancel",
-                                null,
-                                null,
-                                null,
-                                false,
-                                async () =>
+                            null,
```

Comment:
@clmckinley @rd3fa Were any changes made to the following lines? If not, please revert formatting.